src/main/java/ai/rev/speechtotext/CustomVocabulariesClient.java

```diff
@@ -0,0 +1,141 @@
+package ai.rev.speechtotext;
+
+import ai.rev.speechtotext.helpers.ClientHelper;
+import ai.rev.speechtotext.models.CustomVocabulary;
+import ai.rev.speechtotext.models.CustomVocabularyInformation;
+import ai.rev.speechtotext.models.asynchronous.RevAiJobOptions;
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * The CustomVocabularyClient object provides methods to submit custom vocabularies and retrieve
```

Comment:
You could make the class description more generic. Leave it to the individual methods to describe their own purpose.

```suggestion
 * The CustomVocabularyClient object provides methods to interact with the Custom Vocabulary Api
```